src/Features/Core/Portable/PopulateSwitch/PopulateSwitchExpressionHelpers.cs

```diff
@@ -20,6 +20,15 @@ public static ICollection<ISymbol> GetMissingEnumMembers(ISwitchExpressionOperat
             var switchExpressionType = switchExpression?.Type;
 
             var enumMembers = new Dictionary<long, ISymbol>();
+
+            if (switchExpressionType?.OriginalDefinition.SpecialType == SpecialType.System_Nullable_T)
```

Comment:
sorry, i wasn't clear.  you don't even need this 'if' part :)

Reply:
No Problem I’ll remove it.